css/partials/_menu.scss

```diff
@@ -388,6 +388,18 @@
   }
 }
 
+@media screen and (min-width: 568px) and (max-width: 700px) {
```

Comment:
Fix for the "Search" menu item in the main nav going out of it's box in some browsers.